### PR TITLE
Initial changes for 1.23

### PR DESCRIPTION
**Pull request by kschumy**
*Description of changes:*
* Added projects (except etcd, which Jackson already added)
  * Did NOT add Kubernetes patches because we're still figuring those out
* Update version conditional statements throughout

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
checksums will fail. I'll fix them

File: projects/kubernetes/release/1-23/GIT_TAG [hunk 1]
```diff
@@ -0,0 +1 @@
+v0.12.0
```
Comment (kschumy) [inline projects/kubernetes/release/1-23/GIT_TAG:1]: There is a newer version of this ((v0.13.0)[https://github.com/kubernetes/release/tree/v0.13.0]), but I thought we might want to just stay on where we were for 1.22. It wasn't super clear from upstream (https://github.com/kubernetes/kubernetes/blob/v1.23.6/build/common.sh#L94)

Comment (jaxesn) [reply projects/kubernetes/release/1-23/GIT_TAG:1]: I'll look at this one with you

**Review by jaxesn — COMMENTED** [review]
Do we need to add 1-23 to that supported releases file? Or do we not do that until we ready for prod release?

**Comment by kschumy** [comment]
> Do we need to add 1-23 to that supported releases file? Or do we not do that until we ready for prod release?
In the past, we did that as a separate PR, but I added it

**Review by jaxesn — APPROVED** [review]
/unhold
/approve

**Pull request merged**